tests/operators/test_agnostic_load_file.py

```diff
@@ -270,6 +271,39 @@ def test_aql_local_file_to_postgres_no_table_name(self):
             "taxes": 3167.0,
         }
 
+    def test_aql_local_file_to_sqlite_no_table_name(self):
```

Comment:
How important is it to have a SQLite-specific test without a table name? Do we need this type of test for all types of DB?